File: fstp-core/resource/com/purefun/fstp/core/bo/copy/RDSStockBO.java
package com.purefun.fstp.core.bo.copy;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import com.purefun.fstp.core.tool.UUID;

@Entity  
@Table(name="RDS_STOCK") 
public class RDSStockBO  implements BaseBO {	
	@Id
	public String uuid = UUID.createUuid();					
	public long boid = 1001L;							
	public String destination = "fstp.ace.rds.server.stock";	
	public String product_id;
	public String isin;
	public String recv_time;
	public String secu_name_cn;
	public String secu_name_en;
	public String secu_base_id;
	public String exch_type;
	public String secu_type;
	public String secu_sub_type;
	public String currency;
	public double bond_par_value;
	public String last_trade_date;
	public String list_date;
	public int buy_unit;
	public int sell_unit;
	public int trade_low_limit;
	public int trade_high_limit;
	public double pre_close_price;
	public double tick_price;
	public String price_limit_type;
	public double price_high_limit;
	public double price_low_limit;
	public double ex_right_ratio;
	public double dividend_price;
	public String financing_flag;
	public String margin_flag;
	public String secu_status;
	public String memo;	
	
}

File: fstp-core-bo/python/com/purefun/fstp/core/bo/model/RDSStockBO.py
import uuid

class RDSStockBO(object):

    def __init__(self):
        self.uuid = str(uuid.uuid1())
        self.boid = 1001L
        self.destination = "fstp.ace.rds.server.stock"
        self.product_id = ''
        self.isin = ''
        self.recv_time = ''
        self.secu_name_cn = ''
        self.secu_name_en = ''
        self.secu_base_id = ''
        self.exch_type = ''
        self.secu_type = ''
        self.secu_sub_type = ''
        self.currency = ''
        self.bond_par_value = 0
        self.last_trade_date = ''
        self.list_date = ''
        self.buy_unit = 0
        self.sell_unit = 0
        self.trade_low_limit = 0
        self.trade_high_limit = 0
        self.pre_close_price = 0
        self.tick_price = 0
        self.price_limit_type = ''
        self.price_high_limit = 0
        self.price_low_limit = 0
        self.ex_right_ratio = 0
        self.dividend_price = 0
        self.financing_flag = ''
        self.margin_flag = ''
        self.secu_status = ''
        self.memo = ''
